### PR TITLE
move to openzeppelin-solidity package

### PR DESCRIPTION
**Pull request by davidrhodus**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,6 @@
     "truffle": "^4.0.6"
   },
   "dependencies": {
-    "zeppelin-solidity": "1.9.0"
+    "openzeppelin-solidity": "^1.9.0"
```
Comment (libertylocked) [inline package.json]: please pin the exact version

File: contracts/LinniaRecords.sol [hunk 2]
```diff
@@ -1,6 +1,6 @@
 pragma solidity ^0.4.23;
 
-import "node_modules/zeppelin-solidity/contracts/ownership/Ownable.sol";
+import 'openzeppelin-solidity/contracts/ownership/Ownable.sol';
 import "node_modules/zeppelin-solidity/contracts/math/SafeMath.sol";
```
Comment (godfreyhobbs) [inline contracts/LinniaRecords.sol]: What about this one?

Comment (libertylocked) [reply contracts/LinniaRecords.sol]: also inconsistent quote style

**Comment by godfreyhobbs** [comment]
Also please add more description and maybe create a github issue.

File: contracts/LinniaHub.sol [hunk 3]
```diff
@@ -1,6 +1,6 @@
 pragma solidity ^0.4.23;
 
-import "node_modules/zeppelin-solidity/contracts/ownership/Ownable.sol";
+import 'openzeppelin-solidity/contracts/ownership/Ownable.sol';
```
Comment (godfreyhobbs) [inline contracts/LinniaHub.sol]: Be consistent with quotes.

File: test/3_records_test.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ const crypto = require("crypto")
 const eutil = require("ethereumjs-util")
 const multihashes = require("multihashes")
 
-import expectThrow from "zeppelin-solidity/test/helpers/expectThrow"
+import expectThrow from 'openzeppelin-solidity/test/helpers/expectThrow'
```
Comment (godfreyhobbs) [inline test/3_records_test.js]: Quotes...

**Comment by libertylocked** [comment]
Does it not have linter rules on quotation style?

**Pull request merged**